share/patch_postgresql_db_zonemaster_backend_ver_7.0.0.pl

```diff
@@ -20,8 +17,60 @@ sub patch_db {
     ####################################################################
     # TEST RESULTS
     ####################################################################
-    $dbh->do( 'ALTER TABLE test_results ADD COLUMN undelegated integer NOT NULL DEFAULT 0' );
+    eval {
+        $dbh->do( 'ALTER TABLE test_results ADD COLUMN undelegated integer NOT NULL DEFAULT 0' );
+    };
+    print "Error while changing DB schema:  " . $@;
```

Comment:
Shouldn't this be printed only if there are errors?

Reply:
Yes indeed, fixed.